IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_benchmark_SAMV71_XULT/settings/wolfcrypt_benchmark_Debug.jlink

```diff
@@ -0,0 +1,39 @@
+[BREAKPOINTS]
```

Comment:
Or this file. In fact you might be able to not have the settings folder all-together.